jobs/scheduled/enqueue_reminders.rb

```diff
@@ -17,9 +17,18 @@ def skip_enqueue?
 
     def allowed_group_ids
       allowed_groups = SiteSetting.assign_allowed_on_groups.split('|')
+      return [] if allowed_groups.empty?
+
       Group.where(name: allowed_groups).pluck(:id).join(',')
     end
 
+    def membership_condition
+      group_ids = allowed_group_ids
+      condition = 'users.admin OR users.moderator'
+      condition += " OR group_users.group_id IN (#{group_ids})" unless group_ids.empty?
```

Comment:
`if group_ids.present?` is easier to understand than the double negative of `unless / empty`